### DISCUSSION TITLE
Custom CSS Support

### DISCUSSION THREAD

**Discussion by strig** [linuxserver/Heimdall#720] [Ideas]
Is there any way (short of rolling my own container) that allows me to add custom CSS to the dashboard?

**Comment by dasos** [comment]
@strig any reason you closed this? I'd like to do the same. I couldn't immediately see a way to do it.

**Comment by strig** [comment]
@dasos I closed it since I started using a different dashboard, but I'll reopen it for you

**Comment by adripo** [comment]
> 
> 
> @dasos I closed it since I started using a different dashboard, but I'll reopen it for you

What dashboard are you using now?

**Comment by strig** [comment]
@adripo I started using [dashmachine](https://github.com/rmountjoy92/DashMachine)

**Comment by KodeStar** [comment]
Custom CSS and custom js is available from v2.3.0
